MAKE/Makefile.lumi_cpeGnu

```diff
@@ -0,0 +1,90 @@
+CMP = CC
+LNK = CC
+
+# Modules loaded
+# module load PrgEnv-gnu
+# module load gcc/10.3.0
+# module load craype-x86-milan
+# module load papi
```

Comment:
Is that without a purge? Could you put a one-liner version too please?

Reply:
the LUMI workshop recommended procedure was to take a fresh login and do module loads one-by-one. No force purging. Also, modules one by one, otherwise they can sometimes fail to load. But I can add a comment to that effect.

Reply:
Yes I just meant one line for copy-pasting, not a single command.